src/docs/pages/introduction/getting-started/vue.md

```diff
@@ -0,0 +1,109 @@
+```
+
+For more information about this command and Vite, please refer to the [Vite documentation](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).<br />Although this guide assumes your project is using typescript, all examples should also be applicable to javascript projects.
+
+## Install
+
+Add Liquid Oxygen to your project with the package manager of your choice.
+
+```sh
+npm install @emdgroup-liquid/liquid
+```
+
+<ld-notice>
+  Liquid Oxygen works with any package manager (e.g. <code>npm</code>, <code>yarn</code> or <code>pnpm</code>). For simplicity, we use <code>npm</code> in this guide.
+</ld-notice>
+
+### Import stylesheet
+
+All Web Components are loaded togehter with their styles embedded. Therefore we only need to import the global stylesheet for Liquid Oxygen.
+
+Add the following code to your `App.vue` file (or any similar file which is loaded for every page).
+
+```html
+// App.vue
+<script setup lang="ts">
+  import '@emdgroup-liquid/liquid/dist/css/liquid.global.css'
+</script>
+```
+
+## Usage
+
+When adding Liquid Oxygen components to a Vue project, it is crucial to use the Vue bindings. All components are imported from `@emdgroup-liquid/liquid/dist/vue`. The bindings significantly improve Vue compatibility and your developer experience.
+
+Let's have a look at how to add a [LdButton](components/ld-button/) to your project. This examnple also includes a [LdIcon](components/ld-icon/) as it helps you to check if Liquid Oxygen assets are loaded correctly.
+
+```html
+// SampleComponent.vue
+<script lang="ts">
+import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/vue'
+<script>
+
+<template>
+  <LdButton>
+    Click me!
+    <LdIcon name="energy" />
+  </LdButton>
+</template>
+```
+
+When you put this component on a page, you should see a blue button with the text "Click me!" and a lightning bolt icon.
+
+### Events
+
+Liquid Oxygen components aim to work similarly to native HTML elements as much as possible. In most cases, you can expect the same events and behavior from a Liquid Oxygen component and its native equivalent. Custom events are documented on the respective component pages.
+
+Let's take our button from above and add a click handler.
+
+```html
+// SampleComponent.vue
+<script lang="ts">
+  import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/vue'
+  function say(message: string) {
+    alert(message)
+  }
+<script>
+
+<template>
+  <LdButton @click="say('Clicked!')">
+    Click me!
+    <LdIcon name="energy" />
+  </LdButton>
+</template>
+```
+
+Please notice the @ notation of the `@click` prop. This is the expected [Vue convention](https://vuejs.org/guide/essentials/event-handling.html) but differs from the native `onclick` attribute and [documentation](https://www.w3schools.com/tags/ref_eventattributes.asp). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and your code editor's IntelliSense will assist you with that.
```

Comment:
```suggestion
Please notice the `@` notation of the `@click` prop. This is the expected [Vue convention](https://vuejs.org/guide/essentials/event-handling.html), which differs from and should not be confused with native [inline event handlers](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#inline_event_handlers_—_dont_use_these). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and your code editor's IntelliSense will assist you with that.
```

Reply:
Historically w3schools has been seen as a flawed resource (see https://meta.stackexchange.com/questions/87678/discouraging-w3schools-as-a-resource). It has improved by now, but it's still a resource which I personally don't trust, like I trust mdn or or the w3c or whatwg docs (which w3schools has no affiliation with, btw). This event attributes page of the w3schools is a perfect example for being out of date.

TLDR: If you just want to list existing event handlers on elements, the document and window object, use this URL:
https://html.spec.whatwg.org/dev/webappapis.html#event-handlers-on-elements,-document-objects,-and-window-objects